src/ensembl/src/content/app/browser/interstitial/browser-interstitial-instructions/BrowserInterstitialInstructions.tsx

```diff
@@ -0,0 +1,93 @@
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { Step } from 'src/shared/components/step/Step';
+import { ImageButton } from 'src/shared/components/image-button/ImageButton';
+import { Status } from 'src/shared/types/status';
+
+import { ReactComponent as SpeciesSelectorIcon } from 'static/img/launchbar/species-selector.svg';
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as SearchIcon } from 'static/img/sidebar/search.svg';
+
+import styles from './BrowserInterstitialInstructions.scss';
+import { PrimaryButton } from 'src/shared/components/button/Button';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+
+const speciesSelectorButtonClasses = classNames(
+  styles.stepsWrapper,
+  styles.speciesSelectorButton
+);
+
+const BrowserInterstitialInstructions = () => {
+  return (
+    <section className={styles.instructionsPanel}>
+      <div className={styles.instructionsWrapper}>
+        <div className={styles.stepWrapper}>
+          <Step count={1} title={'Find and add a species'} />
+          <div className={styles.description}>
+            <ImageButton
+              className={styles.imageButtonIcon}
+              status={Status.DISABLED}
+              image={SpeciesSelectorIcon}
+            />
+            <div className={styles.iconLabel}>Species Selector</div>
+          </div>
+        </div>
+
+        <div className={styles.stepWrapper}>
+          <Step count={2} title={'Return to this app'} />
+          <div className={styles.description}>
+            <ImageButton
+              className={styles.imageButtonIcon}
+              status={Status.DISABLED}
+              image={BrowserIcon}
+            />
+            <div className={styles.iconLabel}>Genome Browser</div>
+          </div>
+        </div>
+
+        <div className={styles.stepWrapper}>
+          <Step
+            count={3}
+            title={'Use Search or the example links to view a gene or region'}
+          />
+          <div className={styles.searchDescription}>
+            <ImageButton
+              className={styles.searchButtonIcon}
+              status={Status.DISABLED}
+              image={SearchIcon}
+            />
+            <div className={styles.exampleText}>Example gene</div>
+            <div className={styles.exampleText}>Example region</div>
+          </div>
+        </div>
+
+        <div className={speciesSelectorButtonClasses}>
+          <Link to={'/species-selector'}>
```

Comment:
Just like for the `title` attribute value you can wrap this with single/double quotes. Also, makes me wonder if we need any kind of consensus on when not to use them for attributes.